lib/util/fs/index.spec.ts

```diff
@@ -18,7 +20,9 @@ import {
 } from '.';
 
 jest.mock('../../util/exec/env');
+jest.mock('find-up');
 const env = mocked(_env);
+const findUp = mocked(_findUp);
```

Comment:
```suggestion
const { findUp } = mocked({ findUp: _findUp });
```
Let's trick a little bit. 😉 `mocked` is for modules only and don't work well for function imports.

Reply:
@viceice unfortunatelly this workaround didn't help - TypeScript still complains

Reply:
What's the error message? 🤔